### DISCUSSION TITLE
Directly export tokens object.

### DISCUSSION THREAD

**Discussion by koloja** [chakra-ui/panda#3053] [Ideas]
### Description

Id love it if you could export the tokens object from the link at the end of this request to directly address it and other things.
https://github.com/chakra-ui/panda/tree/552b01ca6742b1564dc00cb0058683acde17436b/packages/studio/styled-system/tokens

### Problem Statement/Justification

Dont really have any justification, would just be a nice feature.

### Proposed Solution or API

Add `export tokens;`

**Comment by amandaguthrie** [comment]
I wanted to autogenerate documentation from the panda tokens file and wrote a plugin to add an export statement for tokens during codegen. It might do what you want for now? https://github.com/amandaguthrie/panda-css-presets/blob/main/packages/panda-plugin-add-export/README.md
